src/chains/mainnet/eips.ts

```diff
@@ -0,0 +1,473 @@
+  },
+  {
+    number: 197,
+    title: 'Precompiled contracts for optimal ate pairing check on the elliptic curve alt_bn128',
+    link: 'https://eips.ethereum.org/EIPS/eip-197',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+  },
+  {
+    number: 198,
+    title: 'Big integer modular exponentiation',
+    link: 'https://eips.ethereum.org/EIPS/eip-198',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+    parameters: [
+      {
+        name: 'GQUADDIVISOR',
+        value: 20,
+      },
+    ],
+  },
+  {
+    number: 211,
+    title: 'New opcodes: RETURNDATASIZE and RETURNDATACOPY',
+    link: 'https://eips.ethereum.org/EIPS/eip-211',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+  },
+  {
+    number: 214,
+    title: 'New opcode STATICCALL',
+    link: 'https://eips.ethereum.org/EIPS/eip-214',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+  },
+  {
+    number: 649,
+    title: 'Metropolis Difficulty Bomb Delay and Block Reward Reduction',
+    link: 'https://eips.ethereum.org/EIPS/eip-649',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+  },
+  {
+    number: 658,
+    title: 'Embedding transaction status code in receipts',
+    link: 'https://eips.ethereum.org/EIPS/eip-658',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromByzantium,
+  },
+];
+
+const hardforksFromConstantinople: string[] = getHardforksFrom(MainnetHardfork.Byzantium);
+const constantinopleEIPs: EIP[] = [
+  {
+    number: 145,
+    title: 'Bitwise shifting instructions in EVM',
+    link: 'https://eips.ethereum.org/EIPS/eip-145',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromConstantinople,
+  },
+  {
+    number: 1014,
+    title: 'Skinny CREATE2',
+    link: 'https://eips.ethereum.org/EIPS/eip-1014',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromConstantinople,
+  },
+  {
+    number: 1052,
+    title: 'EXTCODEHASH opcode',
+    link: 'https://eips.ethereum.org/EIPS/eip-1052',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromConstantinople,
+  },
+  {
+    number: 1234,
+    title: 'Constantinople Difficulty Bomb Delay and Block Reward Adjustment',
+    link: 'https://eips.ethereum.org/EIPS/eip-1234',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromConstantinople,
+  },
+  {
+    number: 1283,
+    title: 'Net gas metering for SSTORE without dirty maps',
+    link: 'https://eips.ethereum.org/EIPS/eip-1283',
+    status: EIPState.Final,
+    activeHardforks: hardforksFromConstantinople,
+  },
+];
+
+// The Petersburg hard fork includes the same EIPs as the Constantinople, except the last EIP, EIP-1283.
+const hardforksFromPetersburg: string[] = getHardforksFrom(MainnetHardfork.Petersburg);
+const petersburgEIPs: EIP[] = constantinopleEIPs.slice(0, 3).map((eip) => ({
+  ...eip,
+  activeHardforks: hardforksFromPetersburg,
+}));
```

Comment:
And perhaps we merge pertersburg and constantinople into a single enum entry? Since they went live at the same time on mainnet, I think it's more clear to unify them and show the diff from that hard fork

Reply:
I agree with you. Do you want to keep `Constantinople` or `Petersburg`? The [`ethereum/execution-specs`](https://github.com/ethereum/execution-specs/tree/master/src/ethereum) repository opt for `Constantinople` for example so I would go with that.

Reply:
Done in https://github.com/mds1/evm-diff/pull/50/commits/77d5a182a416f78bd739f64dbe64f8de589c4442, https://github.com/mds1/evm-diff/pull/50/commits/0ff86db7f21cd3dfbfb32f1087dae22c00d4d15b and https://github.com/mds1/evm-diff/pull/50/commits/88c7fcefd78148d751e9308c9164a18c8999bbab